Podverse/PVMediaPlayer.swift

```diff
@@ -19,21 +19,21 @@ enum PlayingSpeed {
         get {
             switch self {
             case .Quarter:
```

Comment:
New Swift 3 standard. enum cases should always start with lowercase. (ie .quarter)